### PR TITLE
feat(recordings): improved performance summary

### PR DESCRIPTION
**Pull request by alexkim205**
## Problem

Screenshot 2023-01-11 at 6 23 51 PM

It is confusing where Interactive, Ready, and Done metrics come from and might not be the performance metrics that are helpful for engineers debugging their website.

## Changes

- [x] Replace Interactive/Ready/Done with FCP, TTI, TBT
- [x] Tooltips with external links explaining each metric and its importance
- [x] Visual indicator of good/poor/bad perf 
- [x] Truncated urls
- [x] Network statuses with new 2xx, 4xx, 5xx filters

Best practices for measuring the performance of a website today are set by Google's Lighthouse metrics, some of which are captured by the performance observer and some which are calculated in house by Google. 

Screenshot 2023-01-11 at 6 25 49 PM

This PR adds code that calculates First Contentful Paint, Time to Interactive, and Total Blocking Time for each page navigation and displays it in a separate performance summary card.

Screenshot 2023-01-11 at 6 37 50 PM

https://user-images.githubusercontent.com/13460330/211939907-f684f4f5-deae-48af-ab9f-21f7976fb50d.mov

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

- [ ] TODO: Add tests for calculating FCP, TTI, and TBT

### CODE REVIEW

File: frontend/src/scenes/session-recordings/player/inspector/v2/components/ItemEvent.tsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ export function ItemEvent({ item, expanded, setExpanded }: ItemEventProps): JSX.
                     <PropertyKeyInfo
                         className="font-medium"
                         disablePopover
+                        disableIcon
```
Comment (benjackwhite) [inline frontend/src/scenes/session-recordings/player/inspector/v2/components/ItemEvent.tsx:21]: Why no icon 😅 ? I thought it looked pretty good to indicate "native" events versus custom ones?

Comment (alexkim205) [reply frontend/src/scenes/session-recordings/player/inspector/v2/components/ItemEvent.tsx:21]: i didn't feel too strongly about it so i'll add it back in :)

**Comment by alexkim205** [comment]
Added 2xx, 4xx, 5xx filters and highlighting based on these response statuses


https://user-images.githubusercontent.com/13460330/212379343-c1580f4c-04b2-4bc3-b797-b7da4ff6c978.mov

**Review by benjackwhite — APPROVED** [review]
Made a couple of changes.
- Removed the minifilters for now. Response status is still experimental so having the UI options there is super confusing for the majority of users who won't have any info there.
- Added the status to the visible list if it is >= 400

**Pull request merged**